CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -90,14 +132,18 @@ static ResizePoolViewModel()
             deallocationOptionValues.AddRange(Enum.GetNames(typeof (ComputeNodeDeallocationOption)));
         }
 
-        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated)
+        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated, string currentAutoScaleFormula)
         {
             this.batchService = batchService;
 
             this.PoolId = poolId;
             this.TargetDedicated = currentDedicated ?? 0;
-            this.DeallocationOptionString = null;
-
+            this.DeallocationOptionString = null;
+            if (string.IsNullOrEmpty(currentAutoScaleFormula))
+                this.IsFixed = true;
+            else
+                this.isAutoScale = true;
+            this.AutoScaleFormula = currentAutoScaleFormula ?? string.Format("$TargetDedicated={0};",currentDedicated ?? 1);
```

Comment:
Why bother setting this to `$TargetDedicated={0}` if `currentAutoScaleFormula` is `null`?  Maybe just do: `this.AutoScaleFormula = currentAutoScaleFormula`?

@paselem What do you think?